### DISCUSSION TITLE
software engineering

### DISCUSSION THREAD

**Discussion by yaadiOroko430** [community/community#131664] [Programming Help]
### Select Topic Area

Question

### Body

let's talk about software engineering

**Comment by teddymillion** [comment]
i'm happy where can we start?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @yaadiOroko430  

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
